exercises/08-exercise.py

```diff
@@ -0,0 +1,132 @@
+'''
+Author: Jorge Diaz
+Exercise 8
+'''
+import uuid
+import pandas as pd
+from databroker.tests.utils import temp_config
+from databroker import Broker
+from databroker.assets.handlers_base import HandlerBase
+
+# this will create a temporary databroker object with nothing in it
+db = Broker.from_config(temp_config())
+
+fc = 7.62939453125e-05
+adc2counts = lambda x: ((int(x, 16) >> 8) - 0x40000) * fc \
+        if (int(x, 16) >> 8) > 0x1FFFF else (int(x, 16) >> 8)*fc
+
+
+class MyHandler(HandlerBase):
+    def __init__(self, resource_path, chunk_size=1024):
+        '''
+        adds the chunks of data to a list for specific file
+
+        Parameters
+        ----------
+        resource_path: str
+            tells the computer where to find the file
+        chunk_size: int (optional)
+            user specifices size of chunk for data, default is 1024
+        '''
+        self.chunks_of_data = []
+        for chunk in pd.read_csv(resource_path, chunksize=chunk_size, 
+                names =['time (s)', 'time (ns)', 'index', 'counts'], 
+                delimiter = " ", header=None):
+            chunk['volts'] = chunk['counts'].apply(adc2counts)
+            chunk['total time (s)'] = chunk['time (s)'] + 1e-9*chunk['time (ns)']
```

Comment:
renamed 'total time (s)' to 'timestamp'. That is how they have it in their data.